packages/react/src/patterns/sub-patterns/ContentSection/ContentSection.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import classNames from 'classnames';
+import { settings as ddsSettings } from '@carbon/ibmdotcom-utilities';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { settings } from 'carbon-components';
+
+const { stablePrefix } = ddsSettings;
+const { prefix } = settings;
+
+/**
+ * ContentArrayGroup Component, for use with content arrays
```

Comment:
Not sure this description is accurate at all.

Reply:
done